main.go

```diff
@@ -36,6 +39,44 @@ func ping(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+/* Handles a call to /books/(exected id here) and redirects depending on the requested action.  */
+func bookById(w http.ResponseWriter, r *http.Request) {
+	method := r.Method
+	if method == http.MethodGet {
+		getBookById(w, r)
+		return
+	} else {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		return
+	}
+}
+
+/* Return the book with that specific ID. */
+func getBookById(w http.ResponseWriter, r *http.Request) {
+	//Isolating ID:
+	justId, _ := strings.CutPrefix(r.URL.Path, "/books/")
```

Comment:
This implementation is a bit fragile. I can show you why, but lets roll with this for now, I don't think it will be a problem for our current use.